Fun Commands/Mini-Games/Dyno-RPG/fester.txt

```diff
@@ -0,0 +1,13 @@
+{delete}
+{silent}
+{require:{&Infected}}
+{!role $1 +Festered}
+**$1** has been inflicted with Fester
+
+**$1** begins to fester. 
+
+**$1**'s skin starts to fall of, piece by rotting piece. The putrid smell of rotted festering flesh fills the air as **$1**'s screams in agony.
```

Comment:
Small grammar issue, 
```suggestion
**$1**'s skin starts to fall of, piece by rotting piece. The putrid smell of rotted festering flesh fills the air as **$1** screams in agony.
```

Reply:
sweet ty. I'll fix that rn. As for the responses I updated them to use the built in selection menus for channel and role requirements. Also updated the README.md file with explanations on how to set up the Dyno RPG commands to get the most out of them =)

Reply:
And done =)